src/stores/LifecycleStore.js

```diff
@@ -61,6 +61,11 @@ class LifecycleStore extends Store {
                 });
                 dis.dispatch(deferredAction);
                 break;
+            case 'on_logged_out':
+                this._state = {
+                    deferred_action: null,
```

Comment:
this duplication feels pretty ming and fragile, surely we should use the same pattern as RoomViewStore and have an INITIAL_STATE variable which gets copied over the top of this._state.